pkg/client/unversioned/services.go

```diff
@@ -38,6 +38,7 @@ type ServiceInterface interface {
 	Delete(name string) error
 	Watch(label labels.Selector, field fields.Selector, opts api.ListOptions) (watch.Interface, error)
 	ProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper
+	SubresourceProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper
```

Comment:
This call should likely be updated to use the new one, whatever it ends up being:
https://github.com/feihujiang/kubernetes/blob/supportSubresourceOfServiceProxy/pkg/controller/podautoscaler/metrics/metrics_client.go#L169